package.json

```diff
@@ -4,7 +4,7 @@
         "packages/*"
     ],
     "scripts": {
-        "build": "turbo run build",
+        "build": "turbo run build --concurrency=5",
```

Comment:
Let’s make this a part of cicd.yaml instead.

Reply:
I've added it to pull-requests.yaml. It looks like cicd.yaml doesn't call the build/test and from a quick look I don't think any other workflows do either

Reply:
Related: https://github.com/solana-labs/wallet-adapter/commit/6f2dfe515224f4c16ebf9803432f5b7b1fc0d93a

Reply:
> It looks like cicd.yaml doesn't call the build/test and from a quick look I don't think any other workflows do either

It does, as a dependency of `publish-packages`.

https://github.com/solana-labs/solana-web3.js/blob/0e113c0e26358bf0ace7b5833f03f84e7a5a38f0/.github/workflows/cicd.yml#L55C11-L55C32

https://github.com/solana-labs/solana-web3.js/blob/0e113c0e26358bf0ace7b5833f03f84e7a5a38f0/package.json#L9

I think the right way to do this is to change `cicd.yml` to read `pnpm publish-packages -- --concurrency=100%` or something. You can test whether that works with `pnpm build -- --concurrency=100%` maybe?

Reply:
Yeah, 100% is best rather than a fixed amount. We do this @ https://github.com/solana-labs/wallet-adapter/blob/ba57f756ec76349c6e46420d387cd706eecd5ad2/package.json#L17